Cargo.toml

```diff
@@ -15,3 +15,6 @@ license = "MIT"
 [dependencies]
 log = "0.4.0"
 glib-sys = { version = "0.9.1", features = ["v2_44"] }
+
+[dev-dependencies]
```

Comment:
This dependency is added only when `cargo test` is run. Forces the tests marked as `#[serial]` to run sequentially.